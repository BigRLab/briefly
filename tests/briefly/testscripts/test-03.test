#> Multiple-thread with multiple targets.

from briefly import *
from testutils import *

objs = Pipeline("test-03")
objs.prop.run_threads = 5

def get_target(n):
  current = objs.phony()
  for i in range(10):
    current = current | test(i + n)
  return current

t1 = get_target(100)
t2 = get_target(200)
t3 = get_target(300)

objs.run(t1, t2, t3)
