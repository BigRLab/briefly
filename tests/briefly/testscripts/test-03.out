>> 100 [True]
>> 101 [True]
>> 102 [True]
>> 103 [True]
>> 104 [True]
>> 105 [True]
>> 106 [True]
>> 107 [True]
>> 108 [True]
>> 109 [True]
>> 200 [True]
>> 201 [True]
>> 202 [True]
>> 203 [True]
>> 204 [True]
>> 205 [True]
>> 206 [True]
>> 207 [True]
>> 208 [True]
>> 209 [True]
>> 300 [True]
>> 301 [True]
>> 302 [True]
>> 303 [True]
>> 304 [True]
>> 305 [True]
>> 306 [True]
>> 307 [True]
>> 308 [True]
>> 309 [True]
